apps/bmslcd/bmsLcd.h

```diff
@@ -0,0 +1,25 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef BMS_LCD_H
+#define BMS_LCD_H
+
+#include "mbed.h"
+#include "TextLCD.h"
+
+class bmsLcd
+{
+    public:
+    bmsLcd(PinName rs, PinName e, PinName d4, PinName d5, PinName d6, PinName d7, LCDType type = TextLCD::LCD16x2);
```

Comment:
This is okay for now. Once we decide on pin lay out, these should become defaults